test/server_tests.js

```diff
@@ -76,7 +78,7 @@ describe('Spotify API', () => {
 			chai.request(server)
 				.get('/spotify/callback')
 				.end((err, res) => {
-					res.should.redirectTo('http://localhost:8888/');
+					res.should.redirectTo('http://localhost:1337/');
```

Comment:
You should construct this from the current port which should be an .env variable as well.